src/extends/ImageInput/index.tsx

```diff
@@ -23,9 +25,24 @@ class ImageInput extends React.Component<Props, State> {
 
   componentDidMount = async () => {};
 
+  onCheckImageName = () => {
+    const { value, onChangeImagePullSecretsRef } = this.props;
+    const imagePullSecret: string[] = [];
+    const imageData = {
+      image: value,
+    };
+    checkImageName(imageData).then((res) => {
+      const data = (res && res.data) || {};
+      if (data.existed && data.sercet) {
+        imagePullSecret.push(data.sercet);
+        onChangeImagePullSecretsRef(imagePullSecret);
+      }
+    });
+  };
+
   render() {
     const { value } = this.props;
-    return <Input {...this.props} defaultValue={value} />;
+    return <Input {...this.props} defaultValue={value} onBlur={this.onCheckImageName} />;
```

Comment:
If the image does not exist, please show the notice message

Reply:
done